websites/S/Shoob/metadata.json

```diff
@@ -0,0 +1,39 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Tranuka",
+		"id": "646006045579411481"
+	},
+    "contributors": [
+		{
+			"name": "Rick",
+			"id": "211982234964590592"
+		},
+		{
+			"name": "<h1>gibigbig",
+			"id": "184087763224756224"
+		},
+		{
+			"name": "theusaf",
+			"id": "193714715631812608"
+		}
+	],
+	"service": "Shoob",
+	"description": {
+		"en": "Waifu Bot?! Anime Card Game by Anime Soul Discord! Connect with all Anime communities today!",
+		"zh_CN": "外福机器人？ ！动漫灵魂不和谐动漫纸牌游戏！立即与所有动漫社区联系！",
+		"ga_IE": "Waifu Bot?! Cluiche Cárta Anime ag Anime Soul Discord! Déan teagmháil le gach pobal Anime inniu!",
+		"nl": "Waifu-bot?! Anime-kaartspel van Anime Soul Discord! Maak vandaag nog contact met alle Anime-gemeenschappen!",
+		"vi_VN": "Bot Waifu ?! Game bài Anime của Anime Soul Discord! Kết nối với tất cả cộng đồng Anime ngay hôm nay!"
+	},
+	"url": "shoob.gg",
```

Comment:
https://animesoul.com/ still exists?

Reply:
yea its not the same pages, it was shoob.gg first V.

Reply:
If the old one still works then don't delete the presence.

Reply:
the Animesoul site will undergo a full remake icon,address and mostly the Name of the Site , I think i had to delete the Presence before it becomes more confusing